plio/models.py

```diff
@@ -158,3 +158,17 @@ def video_update_cache(sender, instance, **kwargs):
     plios = Plio.objects.filter(video_id=instance.id)
     # invalidate saved cache for the plios
     invalidate_cache_for_instances(plios)
+
```

Comment:
we have to be careful of which objects have been included in the response of which other objects through `to_representation` in the serializer.